Tighten checks for "bins" elements in width_bucket(x, bins) (#24103)
prestodb/presto: presto-docs/src/main/sphinx/functions/math.rst

.. function:: width_bucket(x, bins) -> bigint

    Returns the bin number of ``x`` according to the bins specified by the
    array ``bins``. The ``bins`` parameter must be an array of doubles, should not

>     contain ``null`` or non-finite elements and assumed to be in sorted ascending order.

Reviewer: ```suggestion
    contain ``null`` or non-finite elements, and is assumed to be in sorted ascending order.
```

>     The function will generally throw an error if it encounters a ``null`` or non-finite

Reviewer: ```suggestion

    Note: The function returns an error if it encounters a ``null`` or non-finite
```
Suggest adding a line return to left-justify (within the width_bucket entry) a Note that describes the behavior of the function when it encounters a ``null``, and also what and why that error might not be returned.

>     element in ``bins``. However, due to the binary search algorithm some such elements

Reviewer: ```suggestion
    element in ``bins``, but due to the binary search algorithm some such elements
```

    Returns the bin number of ``x`` according to the bins specified by the
    array ``bins``. The ``bins`` parameter must be an array of doubles and is
    assumed to be in sorted ascending order. The function will throw error if

>     encountered a ``null`` element in ``bins``.

Reviewer 2: ```suggestion
    it encounters a ``null`` or non-finite element in ``bins``.
```
